src/biokbase/narrative/jobs/job.py

```diff
@@ -100,7 +98,7 @@ def __init__(self, ee2_state, extra_data=None, children=None):
         if ee2_state.get("job_id") is None:
             raise ValueError("Cannot create a job without a job ID!")
 
-        self._acc_state = ee2_state
+        self._update_state(ee2_state)
```

Comment:
Trying to route all `Job._acc_state` updates through `job._update_state` to get the `job.last_updated` updated appropriately